python/mxnet/gluon/text/vocab.py

```diff
@@ -0,0 +1,325 @@
+        frequency thresholds such as `max_size` and `min_freq`. Keys of `counter`,
+        `unknown_token`, and values of `reserved_tokens` must be of the same hashable type.
+        Examples: str, int, and tuple.
+    max_size : None or int, default None
+        The maximum possible number of the most frequent tokens in the keys of `counter` that can be
+        indexed. Note that this argument does not count any token from `reserved_tokens`. Suppose
+        that there are different keys of `counter` whose frequency are the same, if indexing all of
+        them will exceed this argument value, such keys will be indexed one by one according to
+        their __cmp__() order until the frequency threshold is met. If this argument is None or
+        larger than its largest possible value restricted by `counter` and `reserved_tokens`, this
+        argument has no effect.
+    min_freq : int, default 1
+        The minimum frequency required for a token in the keys of `counter` to be indexed.
+    unknown_token : hashable object, default '<unk>'
+        The representation for any unknown token. In other words, any unknown token will be indexed
+        as the same representation. Keys of `counter`, `unknown_token`, and values of
+        `reserved_tokens` must be of the same hashable type. Examples: str, int, and tuple.
+    reserved_tokens : list of hashable objects or None, default None
+        A list of reserved tokens that will always be indexed, such as special symbols representing
+        padding, beginning of sentence, and end of sentence. It cannot contain `unknown_token`, or
+        duplicate reserved tokens. Keys of `counter`, `unknown_token`, and values of
+        `reserved_tokens` must be of the same hashable type. Examples: str, int, and tuple.
+    embedding : instance or list of instances of `embedding.TokenEmbedding`, default None
+        The embedding to be assigned to the indexed tokens. If a list of multiple embeddings are
+        provided, their embedding vectors will be concatenated for the same token.
+
+
+    Properties
+    ----------
+    embedding : instance of :class:`~mxnet.gluon.text.embedding.TokenEmbedding`
+        The embedding of the indexed tokens.
+    idx_to_token : list of strs
+        A list of indexed tokens where the list indices and the token indices are aligned.
+    reserved_tokens : list of strs or None
+        A list of reserved tokens that will always be indexed.
+    token_to_idx : dict mapping str to int
+        A dict mapping each token to its index integer.
+    unknown_token : hashable object
+        The representation for any unknown token. In other words, any unknown token will be indexed
+        as the same representation.
+
+
+    Examples
+    --------
+    >>> fasttext = text.embedding.create('fasttext', file_name='wiki.simple.vec')
+    >>> text_data = " hello world \n hello nice world \n hi world \n"
```

Comment:
Formatting is broken. http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-10074/10/api/python/gluon/text.html#mxnet.gluon.text.vocab.Vocabulary

Reply:
resolved

Reply:
resolved